ppsci/solver/train.py

```diff
@@ -29,6 +32,26 @@
     from ppsci import solver
 
 
+def _compute_batch_size(
+    input_dict: Dict[str, Union[paddle.Tensor, Sequence[paddle.Tensor]]]
+) -> int:
+    """Compute batch size from given input dict.
+    NOTE: Returned `batch_size` might be inaccurate, but it won't affect the correctness
+    of the training results because `batch_size` is now only used for timing.
+    Args:
+        input_dict (Dict[str, Union[paddle.Tensor, Sequence[paddle.Tensor]]]): Given input dict.
+    Returns:
+        int: Batch size of input dict.
+    """
```

Comment:
```suggestion
    """Compute batch size from given input dict.

    NOTE: Returned `batch_size` might be inaccurate, but it won't affect the correctness
    of the training results because `batch_size` is now only used for timing.

    Args:
        input_dict (Dict[str, Union[paddle.Tensor, Sequence[paddle.Tensor]]]): Given input dict.

    Returns:
        int: Batch size of input dict.
    """
```

Reply:
均已修改